.changeset/polite-shoes-allow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/frontend-app-api': patch
+---
+
+Added the ability to ignore unknown extension config.
```

Comment:
Could you mention how to set the new flag here as well? or just mention that it's a new `flags.allowUnknownExtensionConfig` option for `createApp`